bicepconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"experimentalFeaturesEnabled": {
+		"extensibility": true,
+		"providerRegistry": true,
+		"dynamicTypeLoading": true
+	},
+	"providers": {
+		"radius": "br:biceptypes.azurecr.io/radius:latest",
```

Comment:
How do you expect this to work for a PR?

Reply:
The bicepconfig here is the default for the repo and it shouldn't change (so when you're doing development work, the templates will all compile). These will all use the latest tag. For PRs, we'll have a temporary bicepconfig that's generated with a PR-specific tag for workflows like functional testing, PR builds, etc. That'll be added in a separate PR